app/components/sidebar/logs/Camera.vue

```diff
@@ -1,57 +1,26 @@
 <template>
   <div id="camera-preview">
-      <img :src="url + '?rnd=' + cacheKey">
+      <img v-bind:src="imageSrc">
   </div>
 </template>
 
 <script lang="ts">
-import Vue from "vue";
-import { makeFullUrl } from "../../../store";
-
-var rnd = 0;
-const cameraSocket = new WebSocket("ws://localhost:5001");//connects to websocket in shepherd websocks.py
-// Connection opened
-cameraSocket.addEventListener("open", (event) => {
-  console.log("Connected to camera websocket.")//acknowledge connection to the websocket
-});
-
-/*
-// Listen for messages
-cameraSocket.addEventListener("message", (event) => {
-  if (event.data == 'Camera updated') {
-    rnd++;
-    console.log("Camera updated.");
-    export default {data() { return {newimgurl: imgurl + rnd}}};
-  }
-});
-*/
-
+const ws = new WebSocket("ws://robot.local:5001/");
```

Comment:
I feel like there should be a way to not hardcode robot.local here, just use whatever is currently being used